README.md

```diff
@@ -154,73 +189,90 @@ Check if your Internet connection works
-[en](https://wiki.archlinux.org/index.php/Installation_guide#Connect_to_the_Internet))
-how to make it work or try first
+If you are not connected to the Internet and wish to use Wi-Fi, you'll need to
+identify your wireless network interface and connect using wpa_supplicant. Use
+the following command to list network interfaces and find your wireless network
+device:
+```bash
+$ ip a
+```
 
+Then connect through wpa_supplicant.
+Figure out you <wifi-name> with `ip a`.
 ```bash
-$ dhcpcd
+$ wpa_supplicant -B -i <wifi-name> -c <(wpa_passphrase "Your_SSID" Your_passphrase) && dhclient <wifi-name>
 ```
 
-Now configure the Mirrors, first backup the File
+For more details, consult the [official documentation](https://wiki.archlinux.org/title/Wpa_supplicant) 
+If you encounter issues or prefer a different method for connecting to the Internet, consult the
+[Arch Linux Installation Guide.](https://wiki.archlinux.org/index.php/Installation_guide#Connect_to_the_Internet)
+
+If you're facing issues with obtaining an IP address, you can manually trigger
+the DHCP client as follows:
 
-<details>
-  <summary>Old version</summary>
-  
-  For some reason, the old mirrorlist format isn't used anymore (2021.02.01) and therefore this code won't work anymore...
-  
-  ```bash
-  $ cp /etc/pacman.d/mirrorlist /etc/pacman.d/mirrorlist.bak
-  # XXX is your region e.g. Germany. use cat /etc/pacman.d/mirrorlist to see examples
-  $ grep -E -A 1 ".*XXX.*$" /etc/pacman.d/mirrorlist.bak | sed '/--/d' > /etc/pacman.d/mirrorlist
-  ```
-  
-</details>
+```bash
+$ dhcpcd
+```
+## Configure Mirror List (Optional)
 
+Note: The need to adjust the mirror list can depend on various factors, such as
+your geographical location, network speed requirements, and specific use-cases.
+For most users, the default mirror list should suffice, especially given that
+Arch Linux in 2023 utilizes a ranked system for mirrors. This makes manual
+intervention less necessary. However, if you experience slower-than-expected
+download speeds and are not ( a digital nomad ) frequently changing your
+geographical location, you may want to optimize the mirror list.
+
+Before making any changes, backup your current mirror list:
 ```bash
 $ cp /etc/pacman.d/mirrorlist /etc/pacman.d/mirrorlist.bak
-# XX is your country domain (de for Germany)
-$ grep -E  ".*\.XX.*$" /etc/pacman.d/mirrorlist.bak > /etc/pacman.d/mirrorlist
 ```
 
-Now check if everything went right (You should only see Your entered
-Region)
+To use mirrors from a specific country (replace XX with your country's domain
+code, e.g., de for Germany):
+
+```bash
+$ grep -E  ".*\.XX.*$" /etc/pacman.d/mirrorlist.bak > /etc/pacman.d/mirrorlist
+```
+Check to make sure your mirror list now only contains entries for your chosen country:
 
 ```bash
 $ cat /etc/pacman.d/mirrorlist
 ```
 
-Now download and install the Base System
+## Installing the Base System
```

Comment:
```suggestion
## Installing the base system
```

Reply:
I go with "Title Case" not "Sentence case" in all headings